remoteappmanager/db/orm.py

```diff
@@ -90,6 +93,14 @@ class Accounting(Base):
     application_policy = relationship("ApplicationPolicy")
 
 
+@event.listens_for(Engine, "connect")
+def set_sqlite_pragma(dbapi_connection, connection_record):
```

Comment:
I made the function name `_set_sqlite_pragma` so that sphinx would skip it.  It is the place where future pragma can be added though so I kept the rest of the name